accept/dist/index.js

```diff
@@ -22276,399 +22276,6 @@ module.exports = function btoa(str) {
 }
 
 
-/***/ }),
```

Comment:
can you run `yarn && yarn build` again? this file should not have been modified

Reply:
I just tried, no difference, I have no idea why